advisories/0010.md

```diff
@@ -0,0 +1,49 @@
+### MCR: runc process.cwd and leaked fds container breakout 
+#### Release Date
+2024-01-31
+#### Overview
+Several container breakouts due to internally leaked fds.
+
+Consult the [upstream advisory][upstream] and [snyk][snyk.io] for details, mitigations, workarounds, and patches.
+#### Affected Products
+Mirantis Container Runtime (MCR) <= 23.0.9
+### Vulnerability Information
+#### CVE Identifier
+[CVE-2024-21626][nvd.nist.gov]
+#### CVSSv3.1
+8.6 (High) CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:H
+#### CWEs
+CWE-403, CWE-668
+#### How to check if your environment is vulnerable
+* `containerd.io` package version is equal or lower (<=) than:
+  - 1.6.28~rc.1-1 for DEB package based OSes
+  - 1.6.28-2.1.rc.1.1 for RPM package based OSes
+#### How to fix
+Upgrade MCR packages to the fixed 23.0.9 version or newer
+* `docker-ee*` packages to:
+  - 23.0.9~4-0 for DEB package based OSes
+  - 23.0.9-4 for RPM package based OSes
```

Comment:
Is an RPM package update really required? [Our RPM spec's requirements](https://github.com/Mirantis/ee-packaging/blob/7dab2c633206f2c6fdd27fe7864986c454a45b68/rpm/SPECS/docker-ee.spec#L50) only [constrain the dependency `Version`, not the `Release`](http://ftp.rpm.org/api/4.4.2.2/dependencies.html), in contrast to our deb packages. cc @ameyag

Reply:
In the absence of `Release`, it'll end up clobbering the last 23.0.9 patch.

Reply:
I'm asking if _users_ need to install the bumped docker-ee rpm

Reply:
no, but I would prefer to do this to have more consistent state